server/api/region.go

```diff
@@ -257,6 +257,72 @@ func (h *regionsHandler) GetEmptyRegion(w http.ResponseWriter, r *http.Request)
 	h.rd.JSON(w, http.StatusOK, regionsInfo)
 }
 
+func (h *regionsHandler) GetSizeHistogram(w http.ResponseWriter, r *http.Request) {
+	bound := minRegionHistogramsize
+	if boundStr := r.URL.Query().Get("bound"); boundStr != "" {
+		var err error
+		var bountInput int
+		bountInput, err = strconv.Atoi(boundStr)
+		if err != nil {
+			h.rd.JSON(w, http.StatusBadRequest, err.Error())
+			return
+		}
+		if bound < bountInput {
+			bound = bountInput
+		}
+	}
+	rc := getCluster(r.Context())
+	regions := rc.GetRegions()
+	var histsize *[]int64 = new([]int64)
+	for _, core := range regions {
+		*histsize = append(*histsize, core.GetApproximateSize())
+	}
+	histsizeMap := calHist(bound, histsize)
+	h.rd.JSON(w, http.StatusOK, histsizeMap)
+}
+
+func (h *regionsHandler) GetKeysHistogram(w http.ResponseWriter, r *http.Request) {
```

Comment:
It looks like `GetSizeHistogram`, can we reuse it to avoid redundant code?

Reply:
reuse some code,like this:
```
func calBound(bound int, r *http.Request) (int, error) {
	if boundStr := r.URL.Query().Get("bound"); boundStr != "" {
		bountInput, err := strconv.Atoi(boundStr)
		if err != nil {
			return -1, err
		}
		if bound < bountInput {
			bound = bountInput
		}
	}
	return bound, nil
}
```